### ISSUE TITLE
Upgrade to newer bitnami/postgresql chart version

### ISSUE THREAD

**Issue by lmsurpre** [Alvearie/alvearie-helm#72]
The codecentric/keycloak chart depends on a pretty old version of bitnami/postgresql (postgresql@10.3.13)

To date, I've basically just ignored that and locked in a much more recent version of the bitnami/postgresql chart (e.g. 10.13.x)

Thats worked well til now but, while trying to update to the latest postgresql chart version (10.14.x and 10.15.x), we started hitting the following incompatibility:

> Error: templates/: template: ibm-fhir-server/charts/keycloak/charts/postgresql/templates/statefulset.yaml:404:20: executing "ibm-fhir-server/charts/keycloak/charts/postgresql/templates/statefulset.yaml" at <include "postgresql.readinessProbeCommand" .>: error calling include: template: ibm-fhir-server/charts/postgresql/templates/_helpers.tpl:242:275: executing "postgresql.readinessProbeCommand" at <.Values.containerPorts.postgresql>: nil pointer evaluating interface {}.postgresql

I think we should approach the codecentric crew to see if they are interested in making their chart compatible with newer bitnami/postgresql versions.
If not, we might need to choose between:
A. staying on this older version of the bitnami/postgresql chart (10.13.x); or
B. moving over to the bitnami/keycloak chart for our keycloak dependency

I'm not sure what updates we'd be missing out on...the bitnami charts rev pretty fast but most of the changes don't really seem to matter to us.

**Comment by chgl** [comment]
BTW, the Bitnami postgres chart version 11 now defaults to Postgres 14.2 (plus some other breaking changes).

**Comment by HarrisDePerceptron** [comment]
@lmsurpre so what solution have you arrived at?

**Comment by lmsurpre** [comment]
The latest codecentric keycloak chart now references bitnami postgresql 10.16.2 ... I will try it out shortly and update to that if possible.

**Comment by lmsurpre** [comment]
I messed with it for bit but ran into trouble and opened https://github.com/codecentric/helm-charts/issues/648

**Comment by lmsurpre** [comment]
OK, I was able to work around the codecentric issue by overriding their `keycloak.postgresql.fullname` helper from within our chart.  With this change, we're now at:
* codecentric/keycloak 18.3.0
* bitnami/postgresql 10.16.2

This is the last update I have planned for alvearie/ibm-fhir-server ... from here I think it makes sense to fork this project into LinuxForHealth (where the FHIR Server was moved to after Merative split from IBM).
It probably makes sense to try jumping to KeycloakX while we're at it...I'll open a separate issue for that when the time comes.
